backend/packages/Upgrade/src/api/controllers/validators/StratificationValidator.ts

```diff
@@ -0,0 +1,24 @@
+import { IsNotEmpty, IsObject, IsOptional, IsString } from 'class-validator';
+
+export class FactorStrata {
+  @IsNotEmpty()
+  @IsString()
+  public factor: string;
+
+  @IsObject()
+  public value: Record<string, number>;
```

Comment:
what is "value" here? will it always be a number?

Reply:
`Value` is the object containing all factorValues and the number of its occurrence for a `stratificationFactor`.
Ex:
``` 
{
  factor: FavoriteFood,
  value: {
    burger: 5,
    pizza: 2,
  }
}
```